filebeat/docs/filebeat-options.asciidoc

```diff
@@ -1,544 +1,68 @@
-
-The default setting is disabled.
-
-[float]
-[[scan-order]]
-==== `scan.order`
-
-experimental[]
-
-Specifies whether to use ascending or descending order when `scan.sort` is set to a value other than none. Possible values are `asc` or `desc`.
-
-The default setting is `asc`.
-
-[float]
-==== `harvester_buffer_size`
-
-The size in bytes of the buffer that each harvester uses when fetching a file. The default is 16384.
-
-[float]
-==== `max_bytes`
-
-The maximum number of bytes that a single log message can have. All bytes after `max_bytes` are discarded and not sent.
-This setting is especially useful for multiline log messages, which can get large. The default is 10MB (10485760).
-
-[float]
-[[config-json]]
-==== `json`
-These options make it possible for Filebeat to decode logs structured as JSON messages. Filebeat
-processes the logs line by line, so the JSON decoding only works if there is one JSON object per
-line.
-
-The decoding happens before line filtering and multiline. You can combine JSON decoding with filtering
-and multiline if you set the `message_key` option. This can be helpful in situations where the application
-logs are wrapped in JSON objects, like it happens for example with Docker.
-
-Example configuration:
-
-[source,yaml]
--------------------------------------------------------------------------------------
-json.keys_under_root: true
-json.add_error_key: true
-json.message_key: log
--------------------------------------------------------------------------------------
-
-You must specify at least one of the following settings to enable JSON parsing
-mode:
-
-*`keys_under_root`*:: By default, the decoded JSON is placed under a "json" key in the output document.
-If you enable this setting, the keys are copied top level in the output document. The default is false.
-
-*`overwrite_keys`*:: If `keys_under_root` and this setting are enabled, then the values from the decoded
-JSON object overwrite the fields that Filebeat normally adds (type, source, offset, etc.) in case of conflicts.
-
-*`add_error_key`*:: If this setting is enabled, Filebeat adds a "error.message" and "error.type: json" key in case of JSON
-unmarshalling errors or when a `message_key` is defined in the configuration but cannot be used.
-
-*`message_key`*:: An optional configuration setting that specifies a JSON key on
-which to apply the line filtering and multiline settings. If specified the
-key must be at the top level in the JSON object and the value associated with
-the key must be a string, otherwise no filtering or multiline aggregation will
-occur.
-
-[float]
-==== `multiline`
-
-Options that control how Filebeat deals with log messages that span multiple lines. See <<multiline-examples>> for more information about configuring multiline options.
+//REVIEWERS: I'm using the same example here that's used in the topic about the
+//log input. I wonder if we have a better one that shows a couple of different
+//input types?
```

Comment:
I would say it's good enough, as you're mostly setting generic configs. Maybe @exekias @ph have some more examples available?